cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -180,29 +195,30 @@ func (r *KubeArchiveConfigReconciler) desiredA14eServiceAccount(kaconfig *kubear
 	sa := &corev1.ServiceAccount{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      a14eName,
-			Namespace: kaconfig.Namespace,
+			Namespace: k9eNs,
```

Comment:
With this change, the `kaconfig` parameter is not used.

Reply:
Fixed.